src/bin/common/command.rs

```diff
@@ -0,0 +1,42 @@
+use serde::{Deserialize, Serialize};
+
+// TODO this code is temporary. Due to the limitations of TOML we cannot serialize leftwm::Command
+//      easily. If we replace TOML by JSON/JSON5/YAML we will be able to remove this code and a
+//      bunch of validation in leftwm-check.rs. This requires to deprecate the TOML config file,
+//      thus making a breaking change.
+//
+//      https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=59232ae3a6f902fc3a3a7a09d1d48c80
```

Comment:
We should create an issue to deprecate the toml config and use something else. I love TOML but in this case it really fails... The issue is that it can't serialize enums that has variants with values (struct variants and tuple variants).

Reply:
I am all in favour f this.
I started looking into alternative config languages, when `eww` did their poll/disussion when they moved away from xml.
They wrote their own lisp-like config language called `yuck`, but I don't think that something like this would be a good fit for us.
Back then, I thought `dhall` might be nice, but it might as well be a bit over the top, plus it comes with a ton of crates it depends on.